Gemfile

```diff
@@ -0,0 +1,9 @@
+source 'https://rubygems.org'
+gemspec
+
+gem 'bzip2-ruby', :github => 'chewi/bzip2-ruby', :branch => 'master'
```

Comment:
you need the bzip2 only for "/arf/:id/:cname/:date/:digest/xml". If you manage to drop this api, you will be able to drop this dependency.